app/views/results/index.html.erb

```diff
@@ -26,7 +26,7 @@
     <div class="govuk-grid-column-two-thirds">
       <div class="govuk-grid-row">
         <div class="govuk-grid-column-one-half">
-          <p class="govuk-body search-results__count"><%= number_with_delimiter(@courses.total_count) %> courses found</p>
+          <p class="govuk-body search-results__count"><%= number_with_delimiter(@results_view.total_course_count) %> courses found</p>
```

Comment:
Zero courses issues

Reply:
Yep count needs to be fixed 👍

Reply:
https://github.com/DFE-Digital/find-teacher-training/pull/109/commits/712699839ae0ab984a8ad7897565bc97ff4bc0d7